tests/sentry/relay/snapshots/test_config/test_project_config_with_breakdown/with_metrics.pysnap

```diff
@@ -10,6 +10,7 @@ breakdownsV2:
     - db
     - browser
     - resource
+    - ui
```

Comment:
if this is auto-generated, it should probably be reverted? same for `test_project_config_with_breakdown/without_metrics.pysnap`

Reply:
@marandaneto I'm unsure what you meant by that. Do you not want `ui` span op breakdowns be part of metrics?

Reply:
I meant that I just added a new `ui` value where I found the other operations in the code, I don't know the codebase so I'm not sure if `ui` should be added to those snapshot tests.

Reply:
the same applies for the other snapshop test (test_project_config_with_breakdown/without_metrics.pysnap)
should I revert those 2 files or not?

Reply:
I think it's fine.

@untitaker what do you think?

Reply:
https://github.com/getsentry/sentry/pull/30363#pullrequestreview-842747571